crates/subspace-node/src/cli.rs

```diff
@@ -55,3 +49,21 @@ pub enum Subcommand {
     #[structopt(name = "benchmark", about = "Benchmark runtime pallets.")]
     Benchmark(frame_benchmarking_cli::BenchmarkCmd),
 }
+
+#[allow(missing_docs)]
+#[derive(Debug, StructOpt)]
+pub struct RunCmd {
+    #[allow(missing_docs)]
+    #[structopt(flatten)]
+    pub base: sc_cli::RunCmd,
+}
```

Comment:
Why do we need to wrap `RunCmd` like this right now? It is equivalent to `sc_cli::RunCmd` the way it is written.

Reply:
Gain some consistency with the upstream.

Reply:
Well, they have some extra parameters which we do not, hence it looks weird in our case, but I guess we will add more parameters eventually, so I think this is fine.

Comment would be nice instead of suppression though.

Reply:
Yeah, I thought about that too, we probably at least also add the jaeger option eventually,

Was a leftover in the last commit, comment added.